contrib/boot-contracts-stateful-prop-tests/tests/pox-4/pox-4.stateful-prop.test.ts

```diff
@@ -143,17 +145,22 @@ it("statefully interacts with PoX-4", async () => {
 
   simnet.setEpoch("3.0");
 
+  const successPath = PoxCommands(model.wallets, model.stackers, sut.network);
```

Comment:
Note for the reviewers: The success path was handled by https://github.com/stacks-network/stacks-core/pull/4550 (part of #4579).